js/controller.js

```diff
@@ -295,8 +295,8 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
       if (!this.state.mainVideoInnerWrapper.children('.oo-player-skin').length) {
         this.state.mainVideoInnerWrapper.append("<div class='oo-player-skin'></div>")
       }
-
-      this.state.mainVideoInnerWrapper.attr('style', '');
+      //https://stackoverflow.com/questions/42206645/konvajs-unable-to-preventdefault-inside-passive-event-listener-due-to-target-be
+      this.state.mainVideoInnerWrapper.attr('style', 'touch-action: none');
```

Comment:
Sorry, I didn't notice this line originally. Even though this is not inherited, blocking touches on the main wrapper might have unexpected consequences. We should at least limit this to VR360, but we should look for a different approach if possible.

Reply:
I'm also noticing a bug after pulling these changes. Previously, tapping on the video would should the controls and then you would tap again if you wanted to pause. After these changes, tapping once pauses the video immediately. Could this line be related?